src/main/java/com/apollocurrency/aplwallet/apl/dbmodel/Option.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.apollocurrency.aplwallet.apl.dbmodel;
+
+import com.apollocurrency.aplwallet.apl.Db;
+import org.slf4j.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class Option {
+    private static final Logger LOG = getLogger(Option.class);
+
+    public static String get(String optionName) 
+    {
+        try (Connection con = Db.db.getConnection();
+                Statement stmt = con.createStatement()) {
+            String query = String.format("SELECT * FROM option WHERE name = '%s'", optionName);
```

Comment:
Use PreparedStatement for parameter's injection. SQL query should be represented as constant field. See TwoFactorAuthRepositoryImpl for details.